CMakeLists.txt

```diff
@@ -50,20 +46,33 @@ if(COMPILE_ESP)
             COMMAND openocd -f ${CMAKE_CURRENT_SOURCE_DIR}/tools/openocd/adafruit-esp.cfg -c "program_esp ${CMAKE_BINARY_DIR}/${CMAKE_PROJECT_NAME}.bin 0x10000 verify exit"
     )
 else()
+    #Target to launch local executable
     add_custom_target(
             run
             COMMAND $<TARGET_FILE:${elf_file}>
     )
+
+    #ROS dependencies
+    include(catkin/common)
```

Comment:
Catkin has to be included before the call to `add_executable` otherwise, the executable is not installed correctly in catkin folders.

Reply:
Done